indico/htdocs/css/jquery-ui.css

```diff
@@ -791,7 +791,6 @@ button.ui-button::-moz-focus-inner {
 .ui-widget-content {
 	border: 1px solid #a6c9e2;
 	background: #fcfdfd url(../images/jquery-ui/ui-bg_inset-hard_100_fcfdfd_1x100.png) 50% bottom repeat-x;
-	color: #222222;
```

Comment:
Rats. I didn't think it was a generated file. It's safe to undo this change since this should override it: https://github.com/indico/indico/pull/2668/files#diff-cd5654bb800f0db30dc920d3492c2cccR19